pkg/spec/cluster.go

```diff
@@ -193,6 +193,8 @@ type ClusterStatus struct {
 
 	// Size is the current size of the cluster
 	Size int `json:"size"`
+	// HealthySize is the number of pods passing the liveliness probe, HTTP GET on port 2379
```

Comment:
@xiang90 
Made sense, I'm replying to this comment

> To be more general what about tracking the pods since we're already checking them? Then we can get the names and endpoints both.
healthyPods []*v1.Pod